Source/ChocolateyGui/ViewModels/Controls/LocalSourceControlViewModel.cs

```diff
@@ -108,6 +117,11 @@ public bool CanUpdateAll()
             return this.Packages.Any(p => p.CanUpdate);
         }
 
+        public bool CanExportAll()
+        {
+            return this._exportAll;

```

Comment:
If this is never false, you should be able to just leave of this from the command. Though, thinking about it now, we might want to set _exportAll = false inside ExportAll() so they don't accidentally try and export twice.